sections/dashboard/FuturesPositionsTable/FuturesPositionsTable.tsx

```diff
@@ -332,6 +373,23 @@ const NoPositionsText = styled.div`
 	margin: 20px 0;
 	font-size: 16px;
 	text-align: center;
+	text-decoration: underline;
+`;
+
+const TableNoResults = styled(GridDivCenteredRow)`
```

Comment:
Not sure if we want to, but maybe we can put the styles for the "no results" state in the `Table` component itself, so we only have to pass the message to display, and the CTA text and link route (could be optional).

Reply:
Right now these are actually two different styles since there is a type with bigger padding, so this would be a good improvement

Reply:
I decided to add and export a style from the Table component, so we can import that inside components in case we want more custom styling.